crud/delete.lua

```diff
@@ -21,14 +22,57 @@ local function delete_on_storage(space_name, key)
         return nil, DeleteError:new("Space %q doesn't exist", space_name)
     end
 
-    local tuple = space:delete(key)
-    return tuple
+    -- add_space_schema_hash is false because
+    -- reloading space format on router can't avoid delete error on storage
+    return schema.wrap_box_space_func_result(false, space, 'delete', key)
 end
 
 function delete.init()
    _G._crud.delete_on_storage = delete_on_storage
 end
 
+-- returns result, err, need_reload
+-- need_reload indicates if reloading schema could help
+-- see crud.common.schema.wrap_func_reload()
+local function call_delete(space_name, key, opts)
```

Comment:
I suggest to use e.g. "*_on_router". I really don't like that use place router and storage function in single file but it's another question. "delete_on_storage" says that you perform some logic on "storage-side". "call_delete" says that you call some abstract delete - storage? router? Ok, router, but I need to read source code to understand it.